### ISSUE TITLE
CAN_I_TIMER_CONTAINED and ItemTimers=1

### ISSUE THREAD

**Issue by GladieUO** [Sphereserver/Source-X#1272]
Just so we dont forget.

https://i.gyazo.com/205430b307d206d28f0cfc546124d01a.mp4

**Comment by Jhobean** [comment]
He seem to force timer to 0 to avoid wait 96 sec

**Comment by GladieUO** [comment]
Yes as Jhobean says, everytime it goes to 0 immediately, Its forced so im not waiting. Basically on first 0, it will get stuck at 0. After you move the container twice, it will trigger.

**Comment by GladieUO** [comment]
I think the solution or problem might be little bit more complex. I will try to explain all the issues / behaviours atm.

1.  If you set timer on item inside of players backpack, the item will get stuck at 0, and not trigger @timer
2. If you set timer on equipped item, the @timer will trigger, whoever with the error warning
![Image](https://github.com/user-attachments/assets/58c5b1c9-4d6f-4fb3-ac01-a2e150de3390)

1. What I see as complication is, lets say you have an item waiting for the grabs, on the floor or inside the chest in dungeon, timer is X. Now once you pick the item, and place inside your backpack, the timer is automatically set to -1, which is right, but in this feature case, is wrong.

I'm starting to think that to pin point every piece of potential behaviour to make this work perfectly - using core fixes or more flags/sphere.ini settings, would be more complex, than just removing this feature. 😁

Just imagining that every item placed on ground, and then placed back to backpack, would for example kept the decay timer which is set by default, is bad solution, not mentioning that item with set timer by the user code - lets say timed artifact, would get overwriten by the decay timer after placing on the ground, is just another headache.

Any thoughts?

**Comment by GladieUO** [comment]
excelent now.
